smart-engine/src/main/java/org/smartdata/server/engine/audit/Auditable.java

```diff
@@ -0,0 +1,5 @@
+package org.smartdata.server.engine.audit;
+
```

Comment:
```java
/**
 * Licensed to the Apache Software Foundation (ASF) under one
 * or more contributor license agreements.  See the NOTICE file
 * distributed with this work for additional information
 * regarding copyright ownership.  The ASF licenses this file
 * to you under the Apache License, Version 2.0 (the
 * "License"); you may not use this file except in compliance
 * with the License.  You may obtain a copy of the License at
 *
 *     http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software
 * distributed under the License is distributed on an "AS IS" BASIS,
 * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 * See the License for the specific language governing permissions and
 * limitations under the License.
 */
```
To avoid such remarks in the future, please consider integrating the `license-maven-plugin` into this project. Here is an example of its use: https://github.com/arenadata/prostore/blob/master/pom.xml

Reply:
Added plugin and updated source file headers